src/components/TestimonialCard.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
rename this to `AppTestimonialCard.vue`

Also can we add colours (bg, text and heading) to the cards using the variables defined in `base.css` (you might want to add a new background shade if you want the card to stand out), otherwise this doesn't work out for browsers with prefer dark mode.

Reply:
added a different shade of white and black to the card background-color